copter/source/docs/precision-landing-with-irlock.rst

```diff
@@ -103,7 +103,7 @@ demo is shown below (using an older firmware).
 If the vehicle does behave appropriately, download the dataflash logs
 and examine the PL messages.
 
--  If the "Heal" (meaining health) field is not "1" then there may be a communication issue between the Pixhawk and IR-LOCK sensor.
+-  If the "Heal" (meaning health) field is not "1" then there may be a communication issue between the Pixhawk and IR-LOCK sensor.
```

Comment:
lets try when we make updates to correct things like this also, if in the same sentence:
```suggestion
-  If the "Heal" (meaning health) field is not "1" then there may be a communication issue between the autopilot and IR-LOCK sensor.
```

Reply:
Sure! I've committed this change.